[2024-09-27T20:56:50.918+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T20:56:50.938+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.insert_company_data scheduled__2024-05-19T00:00:00+00:00 [queued]>
[2024-09-27T20:56:50.948+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.insert_company_data scheduled__2024-05-19T00:00:00+00:00 [queued]>
[2024-09-27T20:56:50.950+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-27T20:56:50.965+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): insert_company_data> on 2024-05-19 00:00:00+00:00
[2024-09-27T20:56:50.975+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1594) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T20:56:50.977+0000] {standard_task_runner.py:72} INFO - Started process 1596 to run task
[2024-09-27T20:56:50.978+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_share_prices_with_company_data', 'insert_company_data', 'scheduled__2024-05-19T00:00:00+00:00', '--job-id', '2096', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp0pr5at_j']
[2024-09-27T20:56:50.979+0000] {standard_task_runner.py:105} INFO - Job 2096: Subtask insert_company_data
[2024-09-27T20:56:51.039+0000] {task_command.py:467} INFO - Running <TaskInstance: scrape_and_save_share_prices_with_company_data.insert_company_data scheduled__2024-05-19T00:00:00+00:00 [running]> on host f199b4684a92
[2024-09-27T20:56:51.137+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrape_and_save_share_prices_with_company_data' AIRFLOW_CTX_TASK_ID='insert_company_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-19T00:00:00+00:00'
[2024-09-27T20:56:51.139+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T20:56:51.139+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T20:56:51.140+0000] {logging_mixin.py:190} INFO - Current task name:insert_company_data state:running start_date:2024-09-27 20:56:50.939097+00:00
[2024-09-27T20:56:51.141+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data and current dag run status:running
[2024-09-27T20:56:51.141+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T20:56:54.875+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:57:15.924+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:57:15.950+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:57:37.009+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:57:37.025+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:57:58.090+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:57:58.102+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:58:19.160+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:58:19.170+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:58:40.230+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:58:40.239+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:59:01.356+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:59:01.562+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:59:22.682+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:59:22.691+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T20:59:43.728+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T20:59:43.742+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:00:04.791+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:00:04.801+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:00:25.843+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:00:25.851+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:00:46.938+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:00:46.946+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:01:07.998+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:01:08.007+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:01:29.043+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:01:29.054+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:01:50.126+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:01:50.147+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:02:11.193+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:02:11.204+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:02:32.245+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:02:32.256+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:02:53.287+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:02:53.296+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:03:14.340+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:03:14.348+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:03:35.565+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:03:38.713+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:03:38.714+0000] {job.py:229} INFO - Heartbeat recovered after 12.05 seconds
[2024-09-27T21:03:59.765+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:03:59.773+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:04:20.816+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:04:20.824+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:04:41.861+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:04:41.874+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:05:02.905+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:05:02.912+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:05:23.959+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:05:23.970+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:05:45.370+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:05:45.847+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:05:50.866+0000] {job.py:229} INFO - Heartbeat recovered after 10.74 seconds
[2024-09-27T21:06:06.904+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:06:06.915+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:06:27.909+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:06:27.917+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:06:48.974+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:06:48.984+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:07:10.028+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:07:10.047+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:07:31.084+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:07:31.093+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:07:52.141+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:07:52.164+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:08:13.209+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:08:13.220+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:08:34.253+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:08:34.261+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:08:55.288+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:08:55.295+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:09:16.351+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:09:16.376+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:09:37.441+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:09:37.460+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:09:58.630+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:10:03.851+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:10:24.954+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:10:24.964+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:10:46.019+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:10:46.060+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:11:07.112+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:11:07.122+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:11:28.168+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:11:28.177+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:11:49.233+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:11:49.246+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:12:10.310+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:12:10.329+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:12:31.351+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:12:31.379+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:12:52.432+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:12:52.458+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:13:13.508+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:13:13.521+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:13:34.577+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:13:34.594+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:13:55.645+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:13:55.672+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:14:16.721+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:14:16.733+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:14:37.781+0000] {logging_mixin.py:190} INFO - Error inserting company data: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:14:37.791+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:14:51.623+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2024-09-27T21:14:51.791+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
